docroot/themes/humsci/humsci_basic/patterns/date-stacked-vertical-card/date-stacked-vertical-card.html.twig

```diff
@@ -37,18 +37,16 @@
       <div class="hb-card__title">{{ title }}</div>
     {% endif %}
 
-    {% if time|render|trim or location|render|trim %}
-
-      {% set general_variants = {'time': time, 'location': location} %}
-
+    {% if time|render|trim  %}
       <div class="hb-card__subcontent">
-        {% for key, variant in general_variants %}
-          {% if variant|render|striptags('<drupal-render-placeholder><a>')|trim  %}
```

Comment:
do we need this piece about striptags('<drupal') etc... in there?

Reply:
Yeah I believe the purpose of this was to make sure that if a content field was empty for a piece of content then the pipe separator doesn't show up.

(for example: you could imagine a list of news items where one news item doesn't have an external link but the other ones do)

So that's the case we should test here. I don't know if we need the exact `striptags('<drupal'` solution, but we should handle this somehow.

Reply:
yeah if you all could test for this tomorrow, that would be good to check! I am seeing behavior on my end where there is a pipe after just one item:

<img width="319" alt="Screen Shot 2020-03-10 at 5 01 20 PM" src="https://user-images.githubusercontent.com/12848168/76358989-c8472400-62f0-11ea-9cbb-af0690921c08.png">